src/Spago/Cmd.hs

```diff
@@ -0,0 +1,26 @@
+module Spago.Cmd (getCmdVersion) where
+
+import qualified Spago.Messages as Messages
+import qualified Turtle.Bytes
+import           Spago.Prelude
+import qualified Data.Text      as Text
+import qualified Data.Text.Encoding as Text.Encoding
+import qualified Data.Text.Encoding.Error as Text.Encoding
+import qualified Data.Versions  as Version
+
+-- | Get the semantic version of a command, e.g. purs --version
+getCmdVersion :: Text -> RIO env (Either Text Version.SemVer)
```

Comment:
Having a whole separate module only for this is a bit noisy - we usually hold all these utilities in our custom `Spago.Prelude`, could we move this function there?

Reply:
I did this before, but then I ran into a cyclic dependency issue, that's why I created a separate module